go.mod

```diff
@@ -26,3 +27,7 @@ replace (
 	k8s.io/apimachinery => k8s.io/apimachinery v0.18.6
 	k8s.io/client-go => k8s.io/client-go v0.18.6
 )
+
+replace github.com/crossplane-contrib/terraform-provider-gen => /Users/kasey/src/crossplane-contrib/terraform-provider-gen
+
+replace github.com/crossplane-contrib/terraform-provider-dl => /Users/kasey/src/crossplane-contrib/terraform-provider-dl
```

Comment:
We'll need to drop these here or point to branches that you pushed in forks :+1:

Reply:
✂️